src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/MicrosoftExtensionsLogging/MicrosoftLoggingWrapper.cs

```diff
@@ -71,27 +75,24 @@ private void RecordLogMessage(MethodCall methodCall, MEL.ILogger logger, IAgent
 
             try
             {
-                // We are trying to access this property:
-                // logger.Loggers[0].Logger.ScopeProvider
+                // MEL keeps an array of scope handlers. We can ask one of them for the current scope data.
 
-                // Get the array of Loggers (logger.Loggers[])
```

Comment:
The property we were reading is only available in Microsoft.Extensions.Logging.ConsoleLogger, which won't necessarily be loaded